packages/array-median/src/index.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Selects the k smallest element using [Floyd-Rivest algorithm](https://en.wikipedia.org/wiki/Floyd%E2%80%93Rivest_algorithm)
+ * this is an in-place method.
+ * @private
+ * @param {Array<number>} array
+ * @param {number} k
+ * @return {number} - k smallest element of the array.
+ */
+function select(array, k) {
```

Comment:
that's true, I didn't check that :(, this one is really good https://www.npmjs.com/package/median-quickselect